plugins/tech-insights-backend-module-jsonfc/src/service/JsonRulesEngineFactChecker.ts

```diff
@@ -138,7 +138,7 @@ export class JsonRulesEngineFactChecker
         techInsightChecks,
         Object.values(facts),
       );
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
You can use `isError` from `@backstage/errors` package to remove this `any` cast.

```ts
} catch (e) {
  if (isError(e)) {
    throw new Error(`Failed to run rules engine, ${e.message}`);
  }
  throw e;
}
```

Reply:
Good catch, done!